src/dmd/mars.d

```diff
@@ -2402,13 +2421,14 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             size_t length = argc - i - 1;
             if (length)
             {
-                const(char)[] ext = FileName.ext(arguments[i + 1].toDString());
+                const(char)[] runarg = arguments[i+1].toDString;
```

Comment:
```suggestion
                const(char)[] runarg = arguments[i + 1].toDString();
```

Rationale:
Space around operators, and I remember a maintainer stating a preference for functions like `toDString` having parenthesis to make it obvious they are non-trivial.